containers/concent-api/install-concent-api-python-dependencies.sh.j2

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -e
 
-python="/opt/pyenv/versions/{{ python_version }}.{{ python_patch_version }}/bin/python{{ python_version }}"
+DEFAULT_PYTHON_PATH="/opt/pyenv/versions/{{ python_version }}.{{ python_patch_version }}/bin/python{{ python_version }}"
```

Comment:
Is `DEFAULT_PYTHON_PATH` just your own local variable or something global that other programs (e.g. pyenv or virtualenv) expect? If it's the former, please use lower case.

Also, I think that calling it `default_python` would make it clearer that it's meant to be just a default value for the `python` parameter.